pfdp/css/cta.css

```diff
@@ -0,0 +1,33 @@
+.cta-text {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 39px;
+  text-align: center;
+}
+
+.btn-primary {
+  color: white;
+  background-color: var(--red);
+  height: 80px;
+  width: 407px;
+  text-align: center;
+  border-radius: 53px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 33%;
```

Comment:
😱 
if you want to center a block or inlin-block element, you do it the following way :
`margin: 0 auto;`
here the margin is already in button global css